lib/static/new-ui/components/MainLayout/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import {Gear} from '@gravity-ui/icons';
+import {FooterItem, MenuItem as GravityMenuItem} from '@gravity-ui/navigation';
+import {Icon} from '@gravity-ui/uikit';
+import classNames from 'classnames';
+import React, {ReactNode, useEffect, useState} from 'react';
+import {useSelector} from 'react-redux';
+
+import {UiModeHintNotification} from '@/static/new-ui/components/UiModeHintNotification';
+import styles from '@/static/new-ui/components/MainLayout/index.module.css';
+import {getIsInitialized} from '@/static/new-ui/store/selectors';
+import useLocalStorage from '@/static/hooks/useLocalStorage';
+import {PanelId} from '@/static/new-ui/components/MainLayout/index';
+
+interface FooterProps {
+    visiblePanel: PanelId | null;
+    onFooterItemClick: (item: GravityMenuItem) => void;
+}
+
+export function Footer(props: FooterProps): ReactNode {
+    const isInitialized = useSelector(getIsInitialized);
+    const [isHintVisible, setIsHintVisible] = useState<undefined | boolean>(undefined);
```

Comment:
I dont like explicit undefined here
Lets use "false" or "null" instead of undefined here